app/page.tsx

```diff
@@ -26,38 +29,86 @@ export default function Home() {
 
   const UnderConstructionModal = () => {
     return (
-      <Modal title="Under Construction" open={isModalOpen} onOk={handleOk} onCancel={handleCancel} footer={null}>
+      <Modal
+        title="Under Construction"
+        open={isModalOpen}
+        onOk={handleOk}
+        onCancel={handleCancel}
+        footer={null}
+      >
         <div className="flex items-center flex-col gap-5">
-          <Image
-            src={UnderConstruction}
-            width={200}
-            alt='Under Maintenance'
-          />
-          <div className="text-2xl font-light text-center">Purrintables is under construction. Some links might be broken. In the mean time you can message us over at <a href="https://www.instagram.com/purrintables" target="_blank">Instagram</a> or <a href="https://www.facebook.com/purrintables" target="_blank">Facebook</a>!</div>
+          <Image src={UnderConstruction} width={200} alt="Under Maintenance" />
+          <div className="text-2xl font-light text-center">
+            Purrintables is under construction. Some links might be broken. In
+            the mean time you can message us over at{" "}
+            <a href="https://www.instagram.com/purrintables" target="_blank">
+              Instagram
+            </a>{" "}
+            or{" "}
+            <a href="https://www.facebook.com/purrintables" target="_blank">
+              Facebook
+            </a>
+            !
+          </div>
         </div>
       </Modal>
     );
-  }
+  };
 
   return (
-    <div>
-      <Organism.Header />
+    <main>
       <UnderConstructionModal />
       <div className="mb-10"></div>
-      <div className="container px-5 mx-auto mb-16"><Organism.Carousel /></div>
+      <div className="container px-5 mx-auto mb-16">
+        <Organism.Carousel />
+      </div>
       <Organism.Jumbotron title="Welcome to Purrintables!">
         <div className="space-y-3">
-          <p className="leading-10">Purrintables is a small business based in the Philippines. Here you can support all sorts of our products ranging from <span className="font-bold text-[#df56c0]">prints</span> and <span className="font-bold text-[#df56c0]">stickers</span> to <span className="font-bold text-[#df56c0]">T-shirts</span> and <span className="font-bold text-[#df56c0]">Merch sets</span>.</p>
-          <p className="leading-10">Please read our <span className="font-bold text-[#df56c0]">FAQ</span> pages before making a purchase!</p>
+          <p className="leading-10">
+            Purrintables is a small business based in the Philippines. Here you
+            can support all sorts of our products ranging from{" "}
+            <span className="font-bold text-[#df56c0]">prints</span> and{" "}
+            <span className="font-bold text-[#df56c0]">stickers</span> to{" "}
+            <span className="font-bold text-[#df56c0]">T-shirts</span> and{" "}
```

Comment:
here, too. another idea is to use template literals and move it on top of the code as a variable and not cut the string per line to avoid {" "}